### ISSUE TITLE
code for training a generator

### ISSUE THREAD

**Issue by SkyeLu** [rdevon/DIM#22]
Hi, thank you for your interesting work and releasing the code! Would you kindly release the code bout 

> training a generator by matching to a prior implicitly

?
Thanks a lot!

**Comment by rdevon** [comment]
Yes, and thanks for asking! I'll try to get this out soon, hopefully in 2 weeks time.

**Comment by SkyeLu** [comment]
Thank you! Looking forward to it!

**Comment by rdevon** [comment]
I'm going to leave this open until it's pushed :)
